src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/clients/FeeService.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.infrastructure.clients;
+
+import static uk.gov.hmcts.reform.iacasepaymentsapi.infrastructure.config.FeesConfiguration.LookupReferenceData;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.FeeDto;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.FeeResponse;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.FeeType;
+import uk.gov.hmcts.reform.iacasepaymentsapi.infrastructure.config.FeesConfiguration;
+
+@Service
+@Slf4j
+public class FeeService {
```

Comment:
Not sure if this `FeeService` should be in the `infrastructure` package.  The idea being that anything that touches something external e.g. rest call, db call should go into `infrastructure` package.  Otherwise it should go into the `domain`.  I think the `FeeRegisterApi` class is the actual client.

Reply:
FeeService is now part of the domain